app/models/state_file_az_intake.rb

```diff
@@ -111,11 +111,11 @@ def federal_dependent_count_under_17
   end
 
   def federal_dependent_count_over_17
```

Comment:
I wonder should we rename this method / create a new, given that it now described a situation where the dependent is over 17 but not a senior?

Reply:
Yep, good catch. Will rename!

Reply:
Done